### PR TITLE
Fix/circular loader

### PR DESCRIPTION
**Pull request by pedrohmorais**
## Description
Fix `CircularLoader` on IE and Firefox.

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] IE 11
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile
- [ ] Ux review validation

### CODE REVIEW

File: components/CircularLoader/CircularLoader.jsx [hunk 1]
```diff
@@ -87,10 +96,18 @@ const CircularLoader = props => {
...
   } = props;
 
+  const { cx, cy, r, strokeWidth } = circleProps;
+
   return (
     <Wrapper size={size} gutter={gutter} role="progressbar">
       <Content color={color} viewBox="22 22 44 44">
-        <Circle color={color} />
+        <Circle
+          STROKE-WIDTH={strokeWidth}
```
Comment (alisson-suzigan) [inline components/CircularLoader/CircularLoader.jsx]:
```suggestion
          stroke-width={strokeWidth}
```

Comment (pedrohmorais) [reply components/CircularLoader/CircularLoader.jsx]: This prop is following the material-ui, i have to check if it will not break IE.

Comment (alisson-suzigan) [reply components/CircularLoader/CircularLoader.jsx]: Ok, the ideia is just to move the property name to lower case. Is it a problem?

**Pull request merged**